<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 161, 161);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 103, 103);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="DrawPolygon">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>DrawPolygon</string>
         </property>
         <property name="icon">
          <iconset theme="drive-optical">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearDesk">
         <property name="styleSheet">
          <string notr="true">color: rgb(217, 217, 217);</string>
         </property>
         <property name="text">
          <string>ClearDesk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Information">
         <property name="styleSheet">
          <string notr="true">color: rgb(198, 198, 198);</string>
         </property>
         <property name="text">
          <string>Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Example">
         <property name="styleSheet">
          <string notr="true">color: rgb(225, 225, 225);</string>
         </property>
         <property name="text">
          <string>Example1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Example2">
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 195, 195);</string>
         </property>
         <property name="text">
          <string>Example2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dial">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;
color: rgb(255, 62, 3);
font: 700 italic 11pt &quot;Tw Cen MT&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(39, 79, 118, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 250, 111);</string>
         </property>
         <property name="text">
          <string>N of Vertecies</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>944</width>
         <height>579</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(80, 80, 80);
color: rgb(249, 65, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNewProject">
     <property name="title">
      <string>NewProject</string>
     </property>
     <addaction name="actionAdd_N_of_Vertecies"/>
    </widget>
    <addaction name="menuNewProject"/>
    <addaction name="actionLOAD_from_Json"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFree">
   <property name="text">
    <string>Free</string>
   </property>
   <property name="toolTip">
    <string>Free</string>
   </property>
  </action>
  <action name="actionFixed">
   <property name="text">
    <string>Fixed</string>
   </property>
   <property name="toolTip">
    <string>Fixed</string>
   </property>
  </action>
  <action name="actionAdd_N_of_Vertecies">
   <property name="icon">
    <iconset theme="audio-card">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add N of Vertecies</string>
   </property>
  </action>
  <action name="actionLOAD_from_Json">
   <property name="text">
    <string>LOAD from Json</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
